src/app/components/main-page/main-page.component.html

```diff
@@ -20,12 +20,29 @@
       <a routerLink="/create-post" class="px-2 nav-link-style d-none d-sm-block">
         <fa-icon [icon]="faPlusSquare"></fa-icon>
       </a>
+
+      <div ngbDropdown class="d-inline-block">
+        <button class="img-button border-0 ml-auto" id="post-dropdown" ngbDropdownToggle>
+          <fa-icon [icon]="faEllipsisV"></fa-icon>
+        </button>
+
+        <div ngbDropdownMenu aria-labelledby="post-dropdown" placement="right-bottom">
+          <button ngbDropdownItem (click)="toggleDelete()">Delete Post</button>
+        </div>
+      </div>
     </div>
   </nav>
-
   <div id="searchCollapse" [ngbCollapse]="isSearchCollapsed" class="p-2">
     <input type="text" class="form-control" placeholder="Search Users">
   </div>
+
+  <div class="navbar navbar-light bg-light" *ngIf="eventsService.deleteMode">
+    <span class="navbar-brand">Navbar</span>
```

Comment:
are we going to show the amount of items selected here?

Reply:
Yes implemented it already, just need to push it.